src/directions.rs

```diff
@@ -156,7 +158,7 @@ impl<In> Bearing<In> {
     /// The definition of the azimuthal angle depends on the implementation of [`BearingDefined`]
     /// for `In`.
     ///
-    /// The returned value is always in [-90°, 90°].
+    /// The returned value is always in [-90°, 90°] % 360°
```

Comment:
This doesn't look right. It could suggest that we always return values in 0-90 and 270-360, but that's not true. It's also, I suppose, not true as currently written, given the `uom` property outlined above. Maybe it could be phrased as

```suggestion
    /// The returned value is always in [-90°, 90°] ± N × 360°.
```